Makefile

```diff
@@ -1,51 +1,68 @@
 -include local/Makefile
 
+.PHONY: all deps-go deps-js deps build-go build-server build-cli build-js build build-docker-dev build-docker-full lint-go test-go test-js test run clean
+
 all: deps build
 
 deps-go:
-	go run build.go setup
+	@go run build.go setup
 
 deps-js: node_modules
 
 deps: deps-js
 
 build-go:
-	go run build.go build
+	@echo "build go files"
+	@go run build.go build
 
 build-server:
-	go run build.go build-server
+	@echo "build server"
+	@go run build.go build-server
 
 build-cli:
-	go run build.go build-cli
+	@echo "build in CI environment"
+	@go run build.go build-cli
 
 build-js:
-	yarn run build
+	@echo "build frontend"
+	@yarn run build
```

Comment:
why adding @ ? will result in no output from these.

Reply:
Yeah, I was thinking we might be interested not in the executed commands but what they doing i.e. `echo ...`.
Can revert if you prefer current approach

Reply:
We decided to revert it :)